components/setting/setting.styled.ts

```diff
@@ -1,13 +1,7 @@
 import styled from "styled-components";
+import {SubHeadingButton} from "components/subHeading/subHeadingButton/subHeadingButton.styled";
```

Comment:
```suggestion
import {SubHeadingButton } from "components/subHeading/subHeadingButton";
```

styled component 를 가져오는게 아니라 일반 컴포넌트를 가져와야 하지 않을까요?

Reply:
그러려고 했는데, 일반 컴포넌트를 가져오면 className을 추가해야 하더라고요. 그래서` <SubHeadingButton className="어쩌구"> `무조건 지정을 해야 하는데 이렇게 하는 게 나을까요?

Reply:
좀 생각이 필요했던 이슈였네요.
아래와 같이 컴포넌트를 일반화해서 사용해야하는 거 같아요
본격적으로 디자인 시스템을 이용하는거에요!


![Screen Shot 2022-03-08 at 8 56 34 AM](https://user-images.githubusercontent.com/47841931/157138642-fa180156-9a5a-4455-95a3-287b7af41b78.png)
![Screen Shot 2022-03-08 at 8 58 33 AM](https://user-images.githubusercontent.com/47841931/157138641-1d01102a-69db-4fbb-a946-afb74aefe19c.png)

![Screen Shot 2022-03-08 at 8 58 38 AM](https://user-images.githubusercontent.com/47841931/157138632-53f273a4-9bb8-412c-8435-cc1e9226948d.png)

Reply:
동작은 됐어요! 감사합니다 :) 그런데 이렇게 해야 하는 이유가 뭔가요? 조금만 알려주시면 제가 좀 더 공부해 볼게요!

Reply:
회의 때 이야기해야할거 같은데 짧게 이야기하면,
1. 일반 컴포넌트를 재사용하는게 일반적. styled compoennt 는 스타일을 입히는 수단같은 느낌
2. SubHeadingButton 에 스타일을 결정하는 로직을 넣어야 나중에 수정할때 한번에 수정되니깐 유지보수가 쉬워요! 
  - 예: 여러곳에서 쓰인 SubHeadingButton 의 폰트 크기를 바꾸거나, 빨간색을 좀더 진한 빨간색으로 바꾸는 등

Reply:
https://mui.com/components/buttons/
MUI 참고하시면서 컴포넌트 작성해도 좋을거 같아요